launching/org.eclipse.rcptt.launching.remote/src/org/eclipse/rcptt/launching/remote/Q7RemoteLaunchDelegate.java

```diff
@@ -45,7 +47,7 @@ public void launch(ILaunchConfiguration configuration, String mode,
 		reconnect.setQ7EclPort(AutEventManager.INSTANCE.getPort());
 		reconnect.setId(aut.getId());
 		try {
-			aut.execute(reconnect);
+			aut.execute(reconnect, Q7Launcher.getLaunchTimeout(), monitor);
```

Comment:
This is @basilevs  change. Please rebase commits based on latest master and re create PR